content/docs/1.6.0/references/helm-values.md

```diff
@@ -0,0 +1,232 @@
+| longhornUI.priorityClass | string | `nil` | priorityClass count for longhorn ui |
+| longhornUI.replicas | int | `2` | replica count for longhorn ui |
+| longhornUI.tolerations | list | `[]` | tolerations count for longhorn ui |
+
+### Ingress Settings
+
+| Key | Type | Default | Description |
+|-----|------|---------|-------------|
+| ingress.annotations | string | `nil` | Ingress annotations done as key:value pairs |
+| ingress.enabled | bool | `false` | Set to true to enable ingress record generation |
+| ingress.host | string | `"sslip.io"` |  |
+| ingress.ingressClassName | string | `nil` | Add ingressClassName to the Ingress Can replace the kubernetes.io/ingress.class annotation on v1.18+ |
+| ingress.path | string | `"/"` | If ingress is enabled you can set the default ingress path then you can access the UI by using the following full path {{host}}+{{path}} |
+| ingress.secrets | string | `nil` | If you're providing your own certificates, please use this to add the certificates as secrets |
+| ingress.secureBackends | bool | `false` | Enable this in order to enable that the backend service will be connected at port 443 |
+| ingress.tls | bool | `false` | Set this to true in order to enable TLS on the ingress record |
+| ingress.tlsSecret | string | `"longhorn.local-tls"` | If TLS is set to true, you must declare what secret will store the key/certificate for TLS |
+
+### Openshift Settings
+
+| Key | Type | Default | Description |
+|-----|------|---------|-------------|
+| openshift.enabled | bool | `false` | enable when using openshift |
+| openshift.ui.port | int | `443` | ui port in openshift environment |
+| openshift.ui.proxy | int | `8443` | ui proxy in openshift environment |
+| openshift.ui.route | string | `"longhorn-ui"` | ui route in openshift environment |
+
+### Other Settings
+
+| Key | Default | Description |
+|-----|---------|-------------|
+| annotations | `{}` | Annotations to add to the Longhorn Manager DaemonSet Pods. Optional. |
+| enablePSP | `false` | For Kubernetes < v1.25, if your cluster enables Pod Security Policy admission controller, set this to `true` to ship longhorn-psp which allow privileged Longhorn pods to start |
+| namespaceOverride | `""` | Specify override namespace, specifically this is useful for using longhorn as sub-chart and its release namespace is not the `longhorn-system` |
+| privateRegistry.createSecret | `nil` | Set `true` to create a new private registry secret |
+| privateRegistry.registryPasswd | `nil` | Password used to authenticate to private registry |
+| privateRegistry.registrySecret | `nil` | If create a new private registry secret is true, create a Kubernetes secret with this name; else use the existing secret of this name. Use it to pull images from your private registry |
+| privateRegistry.registryUrl | `nil` | URL of private registry. Leave blank to apply system default registry |
+| privateRegistry.registryUser | `nil` | User used to authenticate to private registry |
```

Comment:
NIT: make this a separate table and mention when users need to care it

Reply:
Fixed, and also mention that these settings are needed when installing Longhorn in air-gap environment
with pointing to the doc 
https://longhorn.io/docs/1.5.1/advanced-resources/deploy/airgap/